src/securityPolicy.c

```diff
@@ -212,6 +212,31 @@ security_policy_t policyForShowDeriveAddress(const addressParams_t* addressParam
 	return _policyForDeriveAddress(addressParams, POLICY_SHOW_BEFORE_RESPONSE);
 }
 
+bool isNetworkUsual(uint32_t networkId, uint32_t protocolMagic)
+{
+	const bool usualNetworkId =
+	        networkId == MAINNET_NETWORK_ID ||
+	        networkId == TESTNET_NETWORK_ID;
+	const bool usualProtocolMagic = protocolMagic == MAINNET_PROTOCOL_MAGIC;
+	return usualNetworkId && usualProtocolMagic;
+}
+
+bool needsMissingCollateralWarning(sign_tx_signingmode_t signingMode, uint32_t numCollaterals)
+{
+	const bool collateralExpected = signingMode == SIGN_TX_SIGNINGMODE_PLUTUS_TX;
+	return collateralExpected && (numCollaterals == 0);
```

Comment:
collateral**s**, to keep the naming consistent.

Reply:
I view it more generally --- collateral inputs are just a particular way of providing collateral for Plutus scripts (the only one existing at present). Calling collateral inputs "collaterals" is useful to keep variable names short in our code, but I don't particularly like it, it's not called like that anywhere else, so I tend to avoid it if I can.